python_course_1604/class_01/tests/test_04.py

```diff
@@ -12,6 +12,7 @@
 
 
 @skip_if_exercize_not_started(exercize)
+@unittest.skip('It brokes coverage.')
```

Comment:
```Python
@unittest.skipIf(
    unittest.skipIf(socket.gethostname() == 'your-host-name'))
```

is equivalent to

```Python
@unittest.skipIf(unittest.skipIf(socket.gethostname() == 'your-host-name'))
```

You can always break lines if you break it between two ( )
What is on the right of @ character is a decorator. I should explain decorators in future classes.
What a decorator expression has to do is returning a function taking one argument (a decorator is a function).

@unitest.slikIf(<some_condition>, <some_reason_description>) returns a function accepting a test function as parameter. Then it can be used like this:

```Python
 @unitest.slikIf(<some_condition>, <some_reason_description>)
 def test_something(self):
     ...
```

I think gethostname function would return the full qualified host name. You can verify it by running this command line:

```Bash
python -c 'import socket; print(socket.gethostname())'
```

On my laptop I have following output (but it changes every day because IT guys in Intel likes to make thinks smarter!): 

fressi-mac03.local

In a Unix machine the host name _should_ never change.

Below code imports socket module.

```Python
import socket
```

socket is a module distributed with Python but it's stuff is not built-in in the language.